var/spack/repos/builtin/packages/ruby-gnuplot/package.py

```diff
@@ -3,20 +3,16 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
-from spack import *
 
-
-class RubyGnuplot(Package):
+class RubyGnuplot(RubyPackage):
     """Utility library to aid in interacting with gnuplot from ruby"""
 
-    homepage = "https://rubygems.org/gems/gnuplot/versions/2.6.2"
+    homepage = "https://github.com/rdp/ruby_gnuplot"
     url      = "https://rubygems.org/downloads/gnuplot-2.6.2.gem"
```

Comment:
I think we should download this package from source (using git) instead of downloading prebuilt gems.

Reply:
Ideally I would like to but I was unable to get this particular package to build from source.